proposals/0014-wave-size-range.md

```diff
@@ -0,0 +1,283 @@
+# Wave Size Range
+
+* Proposal: [0014](0014-wave-size-range.md)
+* Author(s): [Greg Roth](https://github.com/pow2clk)
+* Sponsor: [Greg Roth](https://github.com/pow2clk)
+* Status: **Under Consideration**
+* Planned Version: Shader Model 6.8, validator version 1.8
+
+## Introduction
+
+Shader Model 6.6 included the ability to specify a wave size on a shader entry
+ point in order to indicate that this shader depends on a specific wave size.
```

Comment:
nit: how come some of these lines are indented by one character?

Reply:
I did so to represent when the same sentence was extended to a second line. It keeps with the template recommendation of limiting line length while making clearer where such breaks were made just to keep the width down.